sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/SdkLongHistogram.java

```diff
@@ -28,6 +34,14 @@ private SdkLongHistogram(InstrumentDescriptor descriptor, WriteableMetricStorage
 
   @Override
   public void record(long value, Attributes attributes, Context context) {
+    if (value < 0) {
+      throttlingLogger.log(
```

Comment:
Regarding dropping the point completely, I'm still on the fence here.

1. we *could* update this message to explain how this can break prometheus exporters which it would)
2. OTLP (while violated today) actually handles the situation just fine.  We just can't determine if prometheus-export is ok.

I think this represents the best "first release" option, as at least users understand this could be an issue w/ Prometheus compatibility.   It seems like a restriction we can ease once we figure out a better mechanism for prometheus compatibility here.

Reply:
> I think this represents the best "first release" option

Right. Better to have strict requirements that we ease, since it will be hard / impossible to add restrictions later.